app/components/faq/FaqDetails.js

```diff
@@ -13,7 +13,7 @@ class FaqDetails extends React.Component {
 		header: (
 			<Header
 				title="Information"
-				hasBackButton
+				hasBackButton = { true }
```

Comment:
we don't need this if the value is true, we can omit it and just do `hasBackButton`.  If you run `npm run lint`, it well tell you so